vsphere/panels.libsonnet

```diff
@@ -0,0 +1,1118 @@
+            options: {
+              byField: 'vcenter_host_name',
+              mode: 'outer',
+            },
+          },
+          {
+            id: 'filterFieldsByName',
+            options: {
+              include: {
+                names: [
+                  'vcenter_host_name',
+                  'Value #A',
+                  'Value #B',
+                  'Value #C',
+                  'Value #D',
+                  'Value #E',
+                  'vcenter_vm_name 1',
+                ],
+              },
+            },
+          },
+          {
+            id: 'organize',
+            options: {
+              excludeByName: {},
+              includeByName: {},
+              indexByName: {
+                'Value #A': 2,
+                'Value #B': 3,
+                'Value #C': 4,
+                'Value #D': 5,
+                'Value #E': 6,
+                vcenter_host_name: 0,
+                'vcenter_vm_name 1': 1,
+              },
+              renameByName: {
+                'Value #A': 'CPU utilization',
+                'Value #B': 'Memory utilization',
+                'Value #C': 'Memory ballooned',
+                'Value #D': 'Memory swapped',
+                'Value #E': 'Disk throughput',
+                'Value #F': 'Memory swapped',
+                vcenter_cluster_name: 'Cluster',
+                'vcenter_cluster_name 1': 'Cluster',
+                vcenter_host_name: 'Host',
+                'vcenter_host_name 1': 'Host',
+                vcenter_vm_name: 'Name',
+                'vcenter_vm_name 1': 'Name',
+                'vcenter_vm_name 2': '',
+              },
+            },
+          },
+        ]),
+
+      clusterCPUEffective:
+        barGauge.new(title='Cluster CPU effective')
+        + barGauge.queryOptions.withTargets([
+          t.clusterCPUEffective,
+        ])
+        + barGauge.panelOptions.withDescription('The effective CPU capacity of the cluster.')
+        + barGauge.options.withOrientation('horizontal')
+        + barGauge.standardOptions.withUnit('rotmhz')
+        + barGauge.standardOptions.color.withMode('continuous-YlRd'),
+
+      clusterCPULimit:
+        barGauge.new(title='Cluster CPU limit')
+        + barGauge.queryOptions.withTargets([
+          t.clusterCPULimit,
+        ])
+        + barGauge.panelOptions.withDescription('The available CPU capacity of the cluster.')
+        + barGauge.options.withOrientation('horizontal')
+        + barGauge.standardOptions.withUnit('rotmhz')
+        + barGauge.standardOptions.color.withMode('continuous-YlRd'),
+
+      clusterCPUUtilization:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Cluster CPU utilization',
+          targets=[t.clusterCPUUtilization],
+          description='The CPU utilization percentage of the cluster.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percent')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
```

Comment:
We don't usually use this option unless we're doing min/mean/max, I don't think it adds much here.

Reply:
Goes for all timeseries in this mixin, other than `Top packet error ESXi hosts / 1m`, especially ones w/ the legend on the bottom.

Reply:
Having the display mode as table formats it into a readable form. Otherwise due to how datacenters and vms are named the legend comes out looking like a bunch of gibberish.